source/Outfit.cpp

```diff
@@ -127,6 +140,20 @@ void Outfit::Load(const DataNode &node)
 			++steeringFlareSounds[Audio::Get(child.Token(1))];
 		else if(child.Token(0) == "afterburner effect" && child.Size() >= 2)
 			++afterburnerEffects[GameData::Effects().Get(child.Token(1))];
+		else if(child.Token(0) == "jump effect" && child.Size() >= 2)
+			++jumpEffects[GameData::Effects().Get(child.Token(1))];
```

Comment:
Did you also test what happens if the referenced effect or sound is not present? (I don't expect anything bad to happen; just an empty effect to be played, but let's test this just to be sure.)
Also for the effects below.

Reply:
Tested giving the JD `"jump effect" "this effect does not exist"`. 
When jumping, the game didn't crash or anything, there was just no visible effect. An error was also logged in errors.txt:
`Warning: effect "this effect does not exist" is referred to, but never defined.`